src/lxc/start.c

```diff
@@ -933,8 +933,10 @@ void lxc_end(struct lxc_handler *handler)
 
 	lsm_process_cleanup(handler->conf, handler->lxcpath);
 
-	cgroup_ops->payload_destroy(cgroup_ops, handler);
-	cgroup_ops->monitor_destroy(cgroup_ops, handler);
+	if (cgroup_ops) {
```

Comment:
Oh, this was me having a brainfart. I was thinking that we have check in the callbacks for whether or not cgroup_ops is initialized but that obviously doesn't help when cgroup_ops is NULL...

Reply:
You're missing a SOB line, though!